srcpkgs/linux-base/files/50-bootsize.postinst

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# Kernel hook to check for filled /boot partitions
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+# freespace = blocksize * freeblocks
+freespace=$(( $(stat -f -c %S /boot) * $(stat -f -c %f /boot) ))
+
+# freespace for /boot is <= 100MB
+if [ $freespace -le 104857600 ]; then
+	cat <<-'ENDMSG'
+	 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
I think that message might be noticed more if printed in red or "bold" red

Reply:
It's not hard to add. Too bad we don't have `msg_warn` any other helpers here.